src/Components/Clicker.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import ReactDOM from "react-dom"
+
+class Clicker extends React.Component{
+        
+    state = {
+        counter: 0,
+        increase: 1,
+        cost: 10
+    }
+      
+    playAgain = () => {
+        window.location.reload(false)
```

Comment:
One of the many reason why we use react is to create an app that doesn't need to refresh. Instead of reloading the window, try having the state reset if the button is pressed instead.